internal/models/kubernetessecret/cluster/secretexport/secret_export.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package secret
+
+import (
+	"github.com/go-openapi/swag"
+
+	conditions "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/kubernetessecret/cluster"
```

Comment:
If it is similar to other resources then please move this out as a common utility `statuscondition`.

Reply:
Updated PTAL